packages/data-broker/src/services/dhis/builders/buildEventsFromDhisEventAnalytics.js

```diff
@@ -61,7 +60,7 @@ const buildEvents = (dhisEventAnalytics, dataElementCodes = []) => {
     });
     events.push(event);
   });
-  return events.sort(getSortByKey(METADATA_DIMENSION_TRANSLATION.eventdate));
```

Comment:
Moved this sort to the higher level, which seems fine to me given this function is resposible for 'building' events (not necessarily sorting them) 👍